packages/reporter/src/commands/command.tsx

```diff
@@ -134,6 +134,7 @@ class Command extends Component<Props> {
   render () {
     const { model, aliasesWithDuplicates } = this.props
     const message = model.displayMessage
+    const timeRemaining = model.timeout ? model.timeout - ((new Date()).getTime() - model.wallClockStartedAt.getTime()) : 0
```

Comment:
This is a bit hard to read. I’d usually work with `Date.now` or the `performance.now` results rather than the Date API.